soc/infineon/cat1b/Kconfig.soc

```diff
@@ -0,0 +1,13 @@
+# Copyright (c) 2024 Cypress Semiconductor Corporation (an Infineon company) or
+# an affiliate of Cypress Semiconductor Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+# Family definitions
+config SOC_FAMILY_INFINEON_CAT1
+	bool
```

Comment:
this is also defined here: https://github.com/zephyrproject-rtos/zephyr/blob/80a6a206289537fa220e34fe485b11fccef9ac85/soc/cypress/Kconfig.soc#L7-L8

I know Cypress Semi is acquired  by Infenion, but scatter this in two trees is confusing.
Are there any plans to cleanup this ?
I would prefer things to be aligned, at least as long as those SoC are part of same SoC family.

Reply:
it will cleanup in https://github.com/zephyrproject-rtos/zephyr/pull/70686